cime_config/machines/config_machines.xml

```diff
@@ -3162,6 +3162,117 @@
+          <arg name="ranks_bind">--cpu-bind depth -envall</arg>
+          <arg name="gpu_maps">/soft/tools/mpi_wrapper_utils/gpu_tile_compact.sh</arg>
+       </arguments>
+    </mpirun>
+    <mpirun mpilib="openmpi">
+        <executable>mpirun</executable>
+        <arguments>
+           <arg name="num_tasks">--tag-output -n {{ total_tasks }}</arg>
+           <arg name="tasks_per_node"> --map-by ppr:{{ tasks_per_numa }}:socket:PE=$ENV{OMP_NUM_THREADS} --bind-to hwthread</arg>
+        </arguments>
+    </mpirun>
+    <module_system type="module" allow_error="true">
+       <init_path lang="sh">/soft/packaging/lmod/lmod/init/sh</init_path>
+       <init_path lang="csh">/soft/packaging/lmod/lmod/init/csh</init_path>
+       <init_path lang="python">/soft/packaging/lmod/lmod/init/env_modules_python.py</init_path>
+       <cmd_path lang="sh">module</cmd_path>
+       <cmd_path lang="csh">module</cmd_path>
+       <cmd_path lang="python">/soft/packaging/lmod/lmod/libexec/lmod python</cmd_path>
+       <modules>
+          <command name="purge"></command>
+          <command name="use">/soft/modulefiles</command>
+          <command name="load">spack cmake</command>
+          <command name="use">/soft/restricted/CNDA/updates/modulefiles</command>
+        </modules>
+        <modules compiler="!gnu">
+          <command name="load">oneapi/eng-compiler/2022.12.30.002</command>
+          <!--command name="load">oneapi/eng-compiler/2022.10.15.006</command-->
+          <!--command name="load">mpich/50.2/icc-all-pmix-gpu</command-->
+          <command name="load">intel_compute_runtime/release/22.43.24595.30</command>
+        </modules>
+        <modules compiler="gnu">
+           <command name="unload">spack cmake</command>
+           <command name="load">gcc/10.3.0</command>
+        </modules>
+	<modules>
+	  <command name="load">cray-pals</command>
+          <command name="load">append-deps/default</command>
+        </modules>
+     </module_system>
+     <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
+     <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
+     <environment_variables>
+        <env name="NETCDF_PATH">/lus/gila/projects/CSC249ADSE15_CNDA/software/oneAPI.2022.12.30.003/netcdf</env>
+        <env name="PNETCDF_PATH">/lus/gila/projects/CSC249ADSE15_CNDA/software/oneAPI.2022.12.30.003/pnetcdf</env>
+     </environment_variables>
+     <environment_variables mpilib="mpich" DEBUG="TRUE">
+        <env name="HYDRA_TOPO_DEBUG">1</env>
+     </environment_variables>
+     <environment_variables mpilib="impi">
+        <env name="I_MPI_DEBUG">10</env>
+        <env name="I_MPI_PIN_DOMAIN">omp</env>
+        <env name="I_MPI_PIN_ORDER">spread</env>
+        <env name="I_MPI_PIN_CELL">unit</env>
+     </environment_variables>
+     <environment_variables compiler="oneapi-ifxgpu">
+        <env name="ONEAPI_DEVICE_SELECTOR">level_zero:gpu</env>
+        <env name="ONEAPI_MPICH_GPU">NO_GPU</env>
```

Comment:
So, we don't use GPU-aware MPI on Sunspot because of issues, correct? Remind me ,do we have existing tickets open with Intel on those?

Reply:
yes, we turn it off at the moment

Reply:
we will eventually have this function on in the production run, when it tested fine on sunspot.